### PR TITLE
Ft dashboard template

### PR DESCRIPTION
**Pull request by mansurissa**
#### What does this PR do?
  - Dashboard template
  ---
#### Description of Task to be completed?
- side bar for navigation
- top nav bar for logo and language selection
- test
---
#### How should this be manually tested?
- checkout this branch
- run npm install
- go to http://localhost:4000/signin
- After signing in you will be redirected to dashboard
---
#### What are the relevant trello id?
none

### CODE REVIEW

**Review by samuel776 — APPROVED** [review]
LGTM

**Review by samuel776 — DISMISSED** [review]
LGTM

**Review by 41hulk — DISMISSED** [review]
LGTM

**Review by SIBONIYO — DISMISSED** [review]

File: src/tests/components/logout.test.js [hunk 1]
```diff
@@ -16,8 +12,8 @@ describe('Tests Logout Component ', () => {
...
     fireEvent.click(button);
   });
-  it('should logout when button clicked',()=>{
-    const dispatch = jest.fn();
-    mapDispatchToProps(dispatch).logout();
-  })
+  // it('should logout when button clicked', () => {
+  //   const dispatch = jest.fn();
+  //   mapDispatchToProps(dispatch).logout();
+  // });
```
Comment (raymond42) [inline src/tests/components/logout.test.js]: remove these commented lines

**Review by raymond42 — CHANGES_REQUESTED** [review]
Good work on the PR @mansurissa

File: src/tests/components/logout.test.js (hunk 1 above)
Comment (mansurissa) [reply src/tests/components/logout.test.js]: Done, thanks

**Review by raymond42 — APPROVED** [review]

**Review by samuel776 — APPROVED** [review]

**Pull request merged**